### ISSUE TITLE
Couldn't start config server.

### ISSUE THREAD

**Issue by athiwatc** [zxqfl/TabNine#87]
Config server doesn't start if used on WSL on Windows 10 (Build 1903).

**Comment by zxqfl** [comment]
Hi athiwatc. I just want to double check that the error message says 'unable to start config server' as opposed to 'unable to start web browser'.

Can you run other programs within WSL that bind to TCP ports, e.g. does `nc -l 1234` start successfully?

**Comment by athiwatc** [comment]
Hello @zxqfl, Currently I can not reproduce the error message I got before but I'm sure that it says `config server` as I coped and pasted it into here. Right now I'm getting the message `Couldn't start web browser at http://127.0.0.1:5555/vjewegjzgewgozvcfjsw. See https://tabnine.com/faq#browser_failed`. Which is expected. I will open a new issue if I ever found it again.

**Comment by athiwatc** [comment]
@zxqfl  One additional suggestion is `config_dir` is after `config` so if I only want the `dir` and type `config_dir` the web browser always open up.

**Comment by zxqfl** [comment]
Good point, I'll add an alias for `config_dir` that doesn't start with `config`.
